electricimp/SpiFlashLogger
# CSE-432 and CSE-176

- CSE-432 Test coverage for SPIFlashLogger library
- CSE-176 [SpiFlashLogger] Pull Request #9 should be taken care of

And more changes included:

- Fix codding standard for all file
- Add comments for each method
- Update-improve documentation
- Improve memory usage for read() method
- Fixed an issue with object's start code overflow for a single one logger
- Fixed an issue with start code overflow for too big objects
